tests/tests/es.regexp.exec.js

```diff
@@ -26,3 +26,33 @@ QUnit.test('RegExp#exec capturing groups', assert => {
   // #replace, but here also #replace is buggy :(
   // assert.deepEqual(/(a?)?/.exec('x'), ['', undefined], '/(a?)?/.exec("x") returns ["", undefined]');
 });
+
+QUnit.test('RegExp#exec sticky', assert => {
+  const re = new RegExp('a', 'y');
```

Comment:
We do not polyfill it in IE8- since we can't add accessors. At least, this test should be ignored in engines without accessors.

Reply:
:+1:

I'm also moving it to a separate file.